arc-core/src/arc/input/KeyCode.java

```diff
@@ -169,16 +169,6 @@
     buttonStart(KeyType.key, "Start"),
     buttonSelect(KeyType.key, "Select"),
     buttonMode(KeyType.key, "Button Mode"),
-    numpad0(KeyType.key, "Numpad 0"),
```

Comment:
Keycodes are stored by ordinal - you can't just remove them, as that would break every ID after them.

Reply:
Hmm, just curious, what did you do to fix that after I added keycodes in #5?

Reply:
I moved them to the end of the list.